README.md

```diff
@@ -9,16 +9,38 @@
 
 ## Overview
 
-HealthyHome is an open-source smart home device which monitors the indoors environment conditions, based on a Raspberry Pi.
+HealthyHome is an open-source project centred around a smart home device which monitors the indoor environment conditions of a home.
+
+### Objectives
+
+-   Allow a user to characterise important parameters of their indoor home environment
+-   Provide easy access to both real-time and historical versions of this information
+-   Allow interested parties to recreate this project
+
+### Features
+
+-   Monitors the ambient temperature, barometric pressure, relative humidity, UV index, ambient light levels and concentration of particles in the air
+-   Calculates the indoor air quality (IAQ) index
+-   Provides both real-time and historical measurements
+-   Communicates data to a client (e.g., a smart phone)
+-   Allows user to access real-time and historical data using an application on a client
```

Comment:
Redundant? Kind of delivering the same information as the two above to the end user

Reply:
Removed lines 24, 25 and 26 and summarised real-time and historical data access comments into one line.